content/docs/fragments.md

```diff
@@ -135,8 +135,8 @@ function Glossary(props) {
 }
 ```
 
-`key` is the only attribute that can be passed to `Fragment`. In the future, we may add support for additional attributes, such as event handlers.
+`key`, `Fragment`'e iletilebilecek tek özelliktir. Gelecekte, etkinlik işleyicileri gibi ek özellikler için destek ekleyebiliriz.
```

Comment:
Our common decision for `event handler` is `olay yoneticisi`. So :
>... Gelecekte, olay yöneticileri gibi ek özellikler ....

Please see the TRANSLATION.md file